README.md

```diff
@@ -133,61 +220,17 @@ you can open your IDE preferences (`PHPStorm -> Preferences`) and search for `Fi
 PHP Class Doc Comment, PHP File Header, PHP Interface Doc Comment 
 are those templates that should at least be customized.
 
-16. Use Behat Contexts are divided into `Hooks` - generic app Background, `Setup` specific resource background, `Ui` - specific interaction.
-17. Commit messages should be written (if only it's possible) with the following convention:
-`[Project spec state][Bundle] max 64 characters description in english written in Present Simple.`
-18. If there is an opened issue on Jira for a specific task, your branch should be named `sit_[ISSUE_NUMBER]`. If not, it should be named with the first letter of your name and your surname. In my case (Mikołaj Król) it would be `mkrol`.
-19. Be careful with using static fields.
-20. Be more careful when you think Singleton is something you need in project.
-21. Laravel is not that bad.
-22. Just kidding.
-23. Open source is made by forks if only more than one person is in charge of maintenance of specific package.
-24. We follow http://docs.sylius.org/en/latest/contributing/ contribution standards
-25. `null === $var->getResult($anotherVar)` instead of `$var->getResult($anotherVar) === null`
-26. If that's not obvious yet, be careful with `static`, probably you will never need to use it.
-27. No `/.idea` and other local config files in `.gitignore`. Put them into global gitignore file, read more on https://help.github.com/articles/ignoring-files/#create-a-global-gitignore.
-28. We are working on NIX systems and we don't like Windows nor are we solving it's existance goal and other problems.
-29. Repositories and Entities in public projects should not be defined as `final`. 
-30. Entity fields in public projects (vendors) should be `protected` instad of `private`.
-31. Decorate resource factories with decoration pattern and do not call resource instance with `new` keyword directly. Instead, inject resource factory into constructor and call `createNew()` on it. See `Sylius\Component\Product\Factory\ProductFactory`, `sylius.custom_factory.product` service definition and https://symfony.com/doc/current/service_container/service_decoration.html. The `priority` flag we are starting with equals 1 and is increased by one for each other decoration.
-32. For customizing forms use [Symfony Form Extension](https://symfony.com/doc/2.0/cookbook/form/create_form_type_extension.html).
```

Comment:
Well, this might not be needed as a standard, but it's a valuable protip. Especially to new people or people with less Symfony experience. How about we create a `protips` section?